lms/templates/index.html

```diff
@@ -54,12 +54,17 @@
       % if settings.FEATURES.get('COURSES_ARE_BROWSABLE'):
         <section class="courses">
             <ul class="courses-listing">
-            %for course in courses:
+            %for course in courses[:9]:
               <li class="courses-listing-item">
                 <%include file="course.html" args="course=course" />
               </li>
             %endfor
             </ul>
+            <div class="more-courses">
```

Comment:
The classes/DOM structure here is a bit weird. I'd prefer to see something like the following instead:

````html
<div class="courses-more">
    <a class="courses-more-cta" href="${marketing_link('COURSES')}" > ... </a>
</div>
````